docs/earth-config/earth-config.md

```diff
@@ -24,7 +29,11 @@ git:
 Example:
 
 ```yaml
+global:
+    cache_size_mb: 20000
 git:
+    global:
+        url_instead_of:
```

Comment:
Did you mean to add an example value here?

Reply:
I missed that, thanks. how about something non-github related like:
`url_instead_of: "git@example.com:=https://localmirror.example.com/"`

Reply:
Looks good to me